ffmpeg-processing/workflows/audio-convert.ts

```diff
@@ -0,0 +1,99 @@
+import type { AudioPayload } from "../types.js";
+
+export async function compressAudioWorkflow(input: AudioPayload) {
+	"use workflow";
+
+	const normalized = await normalizeMetadataStep(input);
+	const compressed = await transcodeAndCompressStep(normalized);
+
+	return compressed;
+}
+
+async function normalizeMetadataStep(input: AudioPayload) {
+	"use step";
+
+	const safeFilename =
+		input.filename && input.filename.trim().length > 0
+			? input.filename
+			: "audio.wav";
+
+	if (!input.mimeType.startsWith("audio/")) {
+		throw new Error(
+			`Unsupported MIME type: ${input.mimeType}. Expected audio/*`,
+		);
+	}
+
+	return { ...input, filename: safeFilename };
+}
+
+async function transcodeAndCompressStep(input: AudioPayload) {
```

Comment:
can we use streams instead of buffered data across the wire?

Reply:
Redid moving to sandboxes as vercel-world can't install ffmpeg